### PR TITLE
Created WebGL text tutorial for Python

### PR DESCRIPTION
**Pull request by priyatharsan**
Document for python #973

### CODE REVIEW

File: _posts/python/basic/WebGL/2015-06-30-webgl-text-and-annotations.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+
+<div class="output_area">
+
+<div class="prompt output_prompt">Out[1]:</div>
+
+
+
+<div class="output_html rendered_html output_subarea output_execute_result">
+<iframe id="igraph" scrolling="no" style="border:none;" seamless="seamless" src="https://plot.ly/~priyatharsan/166.embed" height="525px" width="100%"></iframe>
```
Comment (cldougl) [inline _posts/python/basic/WebGL/2015-06-30-webgl-text-and-annotations.html]: @priyatharsan it looks like this plot is set to private:

screen shot 2018-08-08 at 1 05 30 pm

File: _posts/python/basic/WebGL/2015-06-30-webgl-text-and-annotations.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+permalink: python/webgl-text-and-annotations/
+description: How to add webGL based text labels and annotations to plots in python
+name: Python WebGL Text and Annotations | plotly
+has_thumbnail: true
+thumbnail: thumbnail/webgl-text-and-annotations.jpg
+layout: user-guide
+name: WebGL Text and Annotations
+language: python
+title: Python WebGL Text and Annotations | plotly
+display_as: basic
```
Comment (cldougl) [inline _posts/python/basic/WebGL/2015-06-30-webgl-text-and-annotations.html]: can this be added to the style section-- that's where the other annotation/text docs are at this time (https://plot.ly/python/#style-options)

File: _posts/python/basic/WebGL/2015-06-30-webgl-text-and-annotations.html (hunk 1 above)
Comment (priyatharsan) [reply _posts/python/basic/WebGL/2015-06-30-webgl-text-and-annotations.html]: oops! I specified `~notebook_demo/215` . I will check on it.

File: _posts/python/basic/WebGL/2015-06-30-webgl-text-and-annotations.html (hunk 2 above)
Comment (priyatharsan) [reply _posts/python/basic/WebGL/2015-06-30-webgl-text-and-annotations.html]: yep! I will move the doc there

File: _posts/python/basic/WebGL/2015-06-30-webgl-text-and-annotations.html (hunk 1 above)
Comment (priyatharsan) [reply _posts/python/basic/WebGL/2015-06-30-webgl-text-and-annotations.html]: Now I know why. I removed the graph from my plotly account. So thats why it was throwing 404 error. The embed will get updated after plotbot runs I guess. However, after the latest push it shouldn't throw 404 again.

**Comment by priyatharsan** [comment]
Made the changes now  [f76d13c](https://github.com/plotly/documentation/pull/1011/commits/f76d13cfa0fd219b8f6f52f1971416afd41ed82c)

File: _posts/python/style/annotations/webgl-text-and-annotations.ipynb [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    "Plotly's Python library is free and open source! [Get started](https://plot.ly/python/getting-started/) by downloading the client and [reading the primer](https://plot.ly/python/getting-started/).\n",
+    "<br>You can set up Plotly to work in [online](https://plot.ly/python/getting-started/#initialization-for-online-plotting) or [offline](https://plot.ly/python/getting-started/#initialization-for-offline-plotting) mode, or in [jupyter notebooks](https://plot.ly/python/getting-started/#start-plotting-online).\n",
+    "<br>We also have a quick-reference [cheatsheet](https://images.plot.ly/plotly-documentation/images/python_cheat_sheet.pdf) (new!) to help you get started!!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Heatmap with manual annotations"
```
Comment (cldougl) [inline _posts/python/style/annotations/webgl-text-and-annotations.ipynb]: 🐐 capitalize titles (I'd shorten this to Heatmap with Annotations )

Comment (priyatharsan) [reply _posts/python/style/annotations/webgl-text-and-annotations.ipynb]: cool
